packages/skin/src/routes/_index/component/chip/+page.component-browser.js

```diff
@@ -0,0 +1,20 @@
+function toggleAttribute(chip, attribute) {
```

Comment:
Can't we just use the native js toggle? https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute

Reply:
Will change to it, but will need to keep check still (so that if aria-expanded is not set, we dont toggle it)

Reply:
Actually this doesn't work. It only toggles between `aria-pressed` set or unset. We want `aria-expanded="true"` or `aria-expanded="false"` (string values)